### PR TITLE
Make help non-modal and minimazable / maximizable

### PR DESCRIPTION
**Pull request by timlinux**
Note that the non-modal behavior was already there for me. Added minimize/maximize buttons. See #3780

screencapture

### CODE REVIEW

**Comment by timlinux** [comment]
@myarjunar No need to do anything on #3780 from your side ok?

File: safe/gui/tools/help_dialog.py [hunk 1]
```diff
@@ -33,7 +33,9 @@ def __init__(self, parent=None, message=None):
         :type parent: QWidget
         """
 
-        QtGui.QDialog.__init__(self, parent)
+        QtGui.QDialog.__init__(
+            self, parent,
+            flags=Qt.WindowMinimizeButtonHint | Qt.WindowMaximizeButtonHint)
         self.setupUi(self)
         self.setWindowTitle(self.tr('InaSAFE %s Help' % get_version()))
```
Comment (Gustry) [inline safe/gui/tools/help_dialog.py]: Your title is overwrite here @timlinux I think you want to change it.

Comment (timlinux) [inline safe/gui/tools/help_dialog.py]: @Gustry yeah I saw that too - I'm happy to leave the nicer title in the ui file anyway....better than 'Dialog'

Comment (Gustry) [inline safe/gui/tools/help_dialog.py]: Yes better than `Dialog` but I though you wanted to rename your dialog by removing the inasafe version, like we did in the minimum needs calculator

**Comment by Gustry** [comment]
😱You are using windows :D

File: safe/gui/tools/help_dialog.py (hunk 1 above)
Comment (timlinux) [inline safe/gui/tools/help_dialog.py]: Ah - yes maybe that will be a good idea....

**Comment by timlinux** [comment]
Call me @timwindows :-) I think it is good to test a bit with what the masses have....

File: safe/gui/tools/help_dialog.py (hunk 1 above)
Comment (ismailsunni) [inline safe/gui/tools/help_dialog.py]: I think we should have convention on how we set title or text in the dialog. I prefer to do anything beside the static layout / widget in python code. Set text is in python code for me.

Comment (timlinux) [inline safe/gui/tools/help_dialog.py]: @ismailsunni I could not really understand your comment :-P I'm going to go with defining it in the UI for now ok? It should still show up in translation list.

**Comment by Gustry** [comment]
Looks good to me, thanks !

**Pull request merged**